src/components/form/Form.js

```diff
@@ -16,15 +28,20 @@ const Form = ({
       const fieldStructure = field[fieldName];
       return <FormFieldBuilder key={fieldName} fieldName={fieldName} fieldStructure={fieldStructure} />;
     })}
+    <button onClick={onSave} className='button'>Save</button>
```

Comment:
We can use form `onSubmit` and `submit` as button type, so that we can send the form by keyboard